src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -1378,11 +1378,13 @@ Do you wish to continue installing it while keeping the packages that are alread
 If not, {2} needs to uninstall {1} to continue but cannot as they are in use. Try restarting {2} and download {0} again.</value>
   </data>
   <data name="MessageForceInstallOrUninstallUponRestart" xml:space="preserve">
-    <value>Package {0} has one or more dependencies that conflict with the following packages: {1}. Dependency conflicts could cause unintended behavior to occur.
-    
-Do you wish to continue installing it while keeping the packages that are already installed?
+    <value>Package {0} has one or more dependencies that conflict with the following package(s):
+
+{1}
```

Comment:
Does this formatting handle the case where there is more than one package listed one below the other?

Reply:
it does not, I will fix that by adding an option to the `JoinPackageNames` method to join with newlines.